components/event-publisher-proxy/cmd/event-publisher-proxy/main.go

```diff
@@ -59,7 +59,7 @@ func main() {
 	}
 	defer func() {
 		if err := logger.WithContext().Sync(); err != nil {
-			golog.Fatalf("Failed to flush logger, error: %v", err)
+			golog.Printf("Failed to flush logger, error: %v", err)
```

Comment:
Reverting change done here https://github.com/kyma-project/kyma/pull/14712/files, we don't want the EPP to stop if the logger wasn't able to be flushed.